turbo.json

```diff
@@ -1,6 +1,9 @@
 {
   "$schema": "./docs/public/schema.json",
   "pipeline": {
+    "eslint-plugin-turbo#test": {
+      "dependsOn": ["build"]
+    },
```

Comment:
We have "acceptance tests" now for this which have to run after `dist/index.js` is available.